testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapptabs/fragments/MiniAppDisplayFragment.kt

```diff
@@ -73,6 +75,7 @@ class MiniAppDisplayFragment : BaseFragment(), PreloadMiniAppWindow.PreloadMiniA
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        WebView.setWebContentsDebuggingEnabled(true)
```

Comment:
Is it possible to check the variant and set the value ? I know its for the sample app. Just thinking

Reply:
sure, but i was thinking adding a separate PR for the record, but we can do it here, no issues.